_includes/head_custom.html

```diff
@@ -9,7 +9,7 @@
 {% if jekyll.environment == "development" %}
   <script src="{{ '/assets/js/version-selector.js' | relative_url }}"></script>
 {% else %}
-  <script src="{{ '/docs/latest/assets/js/version-selector.js' }}"></script>
+  <script src="{{site.baseurl}}/assets/js/version-selector.js"></script>
```

Comment:
Seems like we are replacing hardcode `/docs/latest` in version selector with `{{site.baseurl}}`. But this doesn't change the path or routing from before. 
Are we plan to update the `baseurl` in the future so eventually the URL pattern would be `docs.opensearch.org/latest/...`

Reply:
Yes, we're just extracting the path into a variable so when we change the url, we can change only the variable in the _config.yml file (and whatever links are hardcoded in the yaml files).